src/index.html

```diff
@@ -11,7 +11,140 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="/https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="group">
+
+        <legend>Personal information</legend>
+
+        <label class="form-field">Surname:
+          <input
+          type="text"
+          name="sname"
+          autocomplete="off"
+          >
```

Comment:
```suggestion
          <input
            type="text"
            name="sname"
            autocomplete="off"
          >
```
fix code style all over your page according to the checklist